tests/acceptance/my-dashboard-test.js

```diff
@@ -0,0 +1,261 @@
+    order2 = make("order", {
+      orderTransportId: order_transport2.get("id"),
+      orderTransport: order_transport2
+    });
+
+    $.mockjaxSettings.matchInRegistrationOrder = false;
+
+    mocks = [];
+
+    const doMock = (modelName, data) => {
+      if (_.isArray(data)) {
+        data = {
+          [pluralize(modelName)]: data
+        };
+      }
+      mocks.push(
+        $.mockjax({
+          url: `/api/v1/${modelName}*`,
+          responseText: data
+        })
+      );
+      mockFindAll(modelName).returns({
+        json: data
+      });
+    };
+
+    doMock("beneficiary", beneficiaries);
+    doMock("gogovan_transport", [
+      make("gogovan_transport").toJSON({
+        includeId: true
+      })
+    ]);
+    doMock("booking_type", [
+      make("booking_type").toJSON({
+        includeId: true
+      })
+    ]);
+    doMock("purpose", [
+      make("purpose").toJSON({
+        includeId: true
+      })
+    ]);
+    doMock("order_transport", [
+      order_transport.toJSON({
+        includeId: true
+      }),
+      order_transport2.toJSON({
+        includeId: true
+      })
+    ]);
+    doMock("order", {
+      orders: [
+        order.toJSON({
+          includeId: true
+        }),
+        order2.toJSON({
+          includeId: true
+        })
+      ],
+      packages: [
+        pkg.toJSON({
+          includeId: true
+        })
+      ],
+      orders_packages: [
+        orders_package.toJSON({
+          includeId: true
+        })
+      ],
+      order_transports: [
+        order_transport.toJSON({
+          includeId: true
+        }),
+        order_transport2.toJSON({
+          includeId: true
+        })
+      ]
+    });
+
+    mocks.push(
+      $.mockjax({
+        url: "/api/v1/auth/current_user_profil*",
+        responseText: userProfile
+      })
+    );
+
+    mocks.push(
+      $.mockjax({
+        url: `/api/v1/orders/${order.get("id")}`,
+        responseText: {
+          order: order.toJSON({
+            includeId: true
+          })
+        }
+      })
+    );
+
+    mocks.push(
+      $.mockjax({
+        url: /\/api\/v1\/images\/\d/,
+        responseText: {
+          image: image.toJSON({
+            includeId: true
+          })
+        }
+      })
+    );
+
+    visit("/");
+  },
+
+  afterEach: function() {
+    $.mockjaxSettings.matchInRegistrationOrder = true;
+    mocks.forEach($.mockjax.clear);
+    Ember.run(App, App.destroy);
+  }
+});
+
+test("should redirect to my_dashboard page", function(assert) {
+  assert.expect(2);
+  visit("/");
+  andThen(function() {
+    click(Ember.$(".menu-3"));
+    andThen(function() {
+      debugger;
+      assert.equal(currentURL(), "/my_orders");
+      assert.equal(Ember.$(".my_dashboard .title").text(), "My Dashboard");
+    });
+  });
+});
+
+test("should redirect to my_account page", function(assert) {
+  assert.expect(3);
+  visit("/");
+  andThen(function() {
+    click(Ember.$(".menu-3"));
```

Comment:
Instead of `.menu-3` can we add id for tab with tab name?

Just a thought.

Reply:
Yeah, we can but currently in the sidebar file, we have all menus like this only, will have to add classes to them also. 
Will make that change. Thanks for pointing out.